tests/acceptance/features/EditBarButtonCest.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
I've tried running this test several times, clearing cache, re-compiling assets and it keeps failing for me. The app seems to be working fine. I also don't know how to manually test this without a button so I'll need some added instructions to test if this is working.

Failing test results:

```./bin/codecept run tests/acceptance/features/EditBarButtonCest.php
Creating cftf_firefox_1 ... done
2018/01/18 17:48:07 Waiting for: tcp://firefox:4444
2018/01/18 17:48:07 Connected to tcp://firefox:4444
2018/01/18 17:48:07 Command finished successfully.
Codeception PHP Testing Framework v2.3.7
Powered by PHPUnit 5.7.26 by Sebastian Bergmann and contributors.

Acceptance Tests (1) -----------------------------------------------------------
⏺ Recording ⏺ step-by-step screenshots will be saved to /var/www/html/tests/_output/
Directory Format: record_5a60ddda4c74b_{testname} ----
Testing acceptance
✖ EditBarButtonCest: See alphabetical list button (1.47s)
--------------------------------------------------------------------------------
DEPRECATION: The "doctrine" service is already initialized, replacing it is deprecated since Symfony 3.3 and will fail in 4.0. /var/www/html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Container.php:204
DEPRECATION: The "doctrine.orm.default_entity_manager" service is already initialized, replacing it is deprecated since Symfony 3.3 and will fail in 4.0. /var/www/html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Container.php:204
⏺ Records saved into: file:///var/www/html/tests/_output/records.html


Time: 7.11 seconds, Memory: 48.00MB

There was 1 failure:

---------
1) EditBarButtonCest: See alphabetical list button
 Test  tests/acceptance/features/EditBarButtonCest.php:seeAlphabeticalListButton
 Step  Fill field "#username","TEST:SUPER_USER:uohara"
 Fail  Field by name, label, CSS or XPath element with '#username' was not found.

Scenario Steps:

 6. $I->fillField("#username","TEST:SUPER_USER:uohara") at tests/_support/Page/LoginLocal.php:52
 5. $I->amOnPage("/login") at tests/_support/Page/LoginLocal.php:51
 4. $I->getLastPassword() at tests/_support/Page/LoginLocal.php:42
 3. $I->getLastUsername() at tests/_support/Page/LoginLocal.php:42
 2. $I->ensureUserExistsWithRole("super_user") at tests/_support/Page/LoginLocal.php:80
 1. $I->amOnPage("/logout") at tests/_support/Page/LoginLocal.php:32


FAILURES!
Tests: 1, Assertions: 0, Failures: 1.
Stopping cftf_firefox_1 ... done
Going to remove cftf_firefox_1
Removing cftf_firefox_1 ... done```